### PR TITLE
Added caption.

### PR DESCRIPTION
**Pull request by steven-rothwell**

### CODE REVIEW

**Comment by steven-rothwell** [comment]
This resolves #2.

**Review by steven-rothwell — COMMENTED** [review]
Looks good.

File: _slides/17-steven-rothwell.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+All your rebases are belong to me.
 
-![skatetocat](https://octodex.github.com/images/skatetocat.png)
+![skatetocat](https://octodex.github.com/images/skatetocat.png)
```
Comment (annakazakova-ddi) [inline _slides/17-steven-rothwell.md:8]: Test **comment**!

**Review by annakazakova-ddi — APPROVED** [review]
Approved!

**Pull request merged**